### ISSUE TITLE
Support circle search (aka. radius query)

### ISSUE THREAD

**Issue by jo-asplin-met-no** [EURODEO/e-soh#60]
It would be useful to get time series sufficiently close to a specified point.

We propose to implement this by splitting the current `spatial_area` in `datastore.proto` into `spatial_polygon` (current behavior) and `spatial_circle`. In the latter case, a _center point_ (lat, lon pair, like 59.91273, 10.74609 for Oslo) and a _radius_ (in kilometers, like 3.84) need to be provided.

So:

```
...

message Polygon { // horizontal polygon area; three or more points
  repeated Point points = 1;
}

message Circle { // horizontal circle area
  Point center = 1;
  double radius = 2;
}

...

message GetObsRequest {
  ...
  // spatial filter
  Polygon spatial_polygon = ...; // if specified, only observations in this polygon may be returned
  Circle spatial_circle = ...; // if specified, only observations in this circle may be returned
   ...
}

...
```

**Comment by jo-asplin-met-no** [comment]
Closed with PR #74

**Comment by jo-asplin-met-no** [comment]
Closed with PR #74
